React/Base/RCTDisplayLink.m

```diff
@@ -16,6 +16,11 @@
 #import "RCTModuleData.h"
 #import "RCTProfile.h"
 
+#if TARGET_OS_OSX // TODO(macOS, https://github.com/microsoft/react-native-macos/issues/533)
+// To compile in Xcode 12 beta 4 on macOS, we need to explicitly pull in the framework to get the definition for CACurrentMediaTime()
+#import <QuartzCore/QuartzCore.h>
+#endif // TODO(macOS, https://github.com/microsoft/react-native-macos/issues/533)
```

Comment:
Is there a reason to not always be explicit about dependencies? I.e. I think we can just unconditionally add this import statement.

Reply:
Perhaps even make sure `QuartzCore` is explicitly being linked against from the respective podspec?

Reply:
Pretty minor impact but to not bloat iOS with deps it already has or doesn't need

Reply:
It's something we've hit plenty of times (see RCTSwitch.m, RCTTextView.m, RCTActivityIndicatorView.m, etc.). iOS does use CACurrentMediaTime as well, but it must be getting included somewhere in the deps that only pulls it in for iOS. I don't see anything suspicious in any podspec about it

Reply:
would probably still err on the side of just including it for iOS then to avoid an odd bug later? It's using import so should be no impact if it is already included indirectly.

Reply:
From offline discussion, looks like the issue is a non-issue on iOS due to RCTPlatformDisplayLink.h conditionally including CADisplayLink for iOS only. Let's see if we can fix this by removing divergence instead of adding another

Reply:
Perfect 👌